INFWS-151 Add a note on cache directory (#2964)
chef/chef-workstation: docs-chef-io/layouts/shortcodes/chef-workstation/ctl_chef_install.md

set. The `Policyfile.lock.json` file may be used to install the locked
policy set to other machines and may be pushed to a policy group on the
Chef Infra Server to apply that policy to a group of nodes that are
under management by Chef.

<div class="admonition-note">

<p class="admonition-note-title">Note</p>

<div class="admonition-note-text">

> By default, the cookbook cache is located in `~/.chef-workstation`
> on macOS / Linux and in `%LOCALAPPDATA%\chef-workstation` on Windows.
> This can be changed by defining the `CHEF_WORKSTATION_HOME` environment
> variable and setting its value to the desired cache directory.

Reviewer: ```suggestion
By default, the cookbook cache is located in `~/.chef-workstation`
on macOS and Linux, and in `%LOCALAPPDATA%\chef-workstation` on Windows.
You can change the cache location by defining the `CHEF_WORKSTATION_HOME` environment
variable and setting its value to the desired cache directory.
```

Reviewer: Do we have documentation somewhere for setting the `CHEF_WORKSTATION_HOME` environment
variable? I'm not finding anything. We should have something somewhere telling people how to do this first.

Author: No existing documentation. This is the intent of the note here, this environment variable primarily caters to cache used by `chef-cli`.

Author: I can add a separate note to help users set an environment variable in general.

Reviewer: Is `CHEF_WORKSTATION_HOME` just set in a bashrc or zshrc file? If so we could probably just do something like:

> Set the desired location of the cache directory by setting the `CHEF_WORKSTATION_HOME` environment variable in your `.bashrc` or `zshrc` file. For example, `CHEF_WORKSTATION_HOME="~/.workstation"`.

Author: yes, that is apt in macOS and Linux, on Windows the technique is based on adding via settings or from terminal by using [setx](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/setx). i'll modify and push a commit.
